cartridge/graphql/parse.lua

```diff
@@ -278,10 +292,20 @@ local graphQL = P {
   directive = '@' * name * maybe('arguments') / cDirective,
   directives = ws * list('directive', 1) * ws,
 
-  variableDefinition = ws * variable * ws * ':' * ws * _'type' * (ws * '=' * _'value') ^ -1 * comma * ws / cVariableDefinition,
+  variableDefinition = ws
```

Comment:
I'm not sure that this part improves readability (If "readability" is applicable for lpeg patterns at all...)

Reply:
It suppresses luacheck warning about too long line